hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -503,27 +503,14 @@ public void init(@NonNull final Platform platform, @NonNull final NodeId nodeId)
                         logger.info("Hederanode#{} is ACTIVE", nodeId);
                         startGrpcServer();
                     }
-                    case BEHIND -> {
-                        logger.info("Hederanode#{} is BEHIND", nodeId);
-                        if (wasActive) shutdownGrpcServer();
-                    }
-                    case FREEZE_COMPLETE -> logger.info("Hederanode#{} is in FREEZE_COMPLETE", nodeId);
-                    case REPLAYING_EVENTS -> logger.info("Hederanode#{} is REPLAYING_EVENTS", nodeId);
-                    case STARTING_UP -> logger.info("Hederanode#{} is STARTING_UP", nodeId);
-                    case CATASTROPHIC_FAILURE -> {
-                        logger.info("Hederanode#{} is in CATASTROPHIC_FAILURE", nodeId);
-                        if (wasActive) shutdownGrpcServer();
-                    }
-                    case CHECKING -> {
-                        logger.info("Hederanode#{} is CHECKING", nodeId);
-                        if (wasActive) shutdownGrpcServer();
-                    }
-                    case OBSERVING -> logger.info("Hederanode#{} is OBSERVING", nodeId);
-                    case FREEZING -> {
-                        logger.info("Hederanode#{} is FREEZING", nodeId);
+
+                    case FREEZE_COMPLETE, REPLAYING_EVENTS, STARTING_UP, OBSERVING, RECONNECT_COMPLETE -> logger.info(
+                            "Hederanode#{} is {}", nodeId, platformStatus.name());
+
+                    case CATASTROPHIC_FAILURE, CHECKING, FREEZING, BEHIND -> {
```

Comment:
I don't think we should be shutting down the server if it's in `CHECKING` state, as that state is recoverable and transient.  Even `BEHIND` can recover (the node will reconnect).
If we do not shut down when `FREEZING` then we can probably re-enable some tests that send a `FREEZE` transaction (followed by cancelling the freeze).

Thoughts?

Reply:
Makes sense to me but this is way out of my depth. cc: @rbair23

Reply:
Hmm Neeha at least when `hedera.profiles.active=PROD`, I can't see a reason to keep the gRPC server open on either `FREEZING` or `FREEZE_COMPLETE`. 

Maybe when `hedera.profiles.active=DEV`, there is some reason not to shut down gRPC server as @jsync-swirlds mentioned?

Don't have strong intuition here. 🤔

Reply:
Currently even dev and test run with the PROD profile.
When `FREEZING` the node probably should still stay active until `FREEZE_COMPLETE`; mostly because it could be cancelled before the freeze is completed (we have existing tests that start a freeze, verify `FREEZING` state, then cancel the freeze), and shutting down the server before then generally breaks the ability to cancel the freeze process.

Reply:
What is the verdict here, guys? When should we shut down the GRPC server and when should we just log a message?

Reply:
I believe we should only shut down the gRPC server on FREEZE_COMPLETE or CATASTROPHIC_FAILURE to preserve pre-existing expectations and behaviors.
We should await confirmation from @Neeharika-Sompalli and @tinker-michaelj, however.

Reply:
yeah I agree with @jsync-swirlds. That seem right to me.

Reply:
Same, seems reasonable to limit shutdown to just `FREEZE_COMPLETE` and `CATASTROPHIC_FAILURE`.

Reply:
Updated 👍 Thank you, guys!